_hooks/pre-commit

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+# A git hook script that runs the same checks that are executed on Travis to
+# verify your commits will not fail. Bypass it with the --no-verify option to
+# git commit
+#
+# usage: make a symlink to this file,
+# e.g., rm -rf .git/hooks/ && ln -s -f ../_hooks .git/hooks
+
+#######################################
+# Error function to display error message to end users.
+# Arguments:
+#   Error message description
+#######################################
```

Comment:
Dang, I coulda swore I took that out as I decided it was overkill, shoulda looked the files over again after committing.

Fixed now in https://github.com/m-lab/m-lab.github.io/pull/81/commits/eabd7618abc1915efa0d31b50e1c9b29e56e6078